editor/app/crashes/[record_locator]/page.tsx

```diff
@@ -24,16 +24,36 @@ import { peopleRelatedRecordCols } from "@/configs/peopleRelatedRecordTable";
 import { Crash } from "@/types/crashes";
 import CrashRecommendationCard from "@/components/CrashRecommendationCard";
 import CrashSwapAddressButton from "@/components/CrashSwapAddressButton";
+import { useKeyboardShortcut } from "@/utils/shortcuts";
 
 const typename = "crashes";
 
+// Lookup object that maps key shortcuts to the associated DOM element id to scroll to
+const shortcutKeyLookup = {
+  A: "address",
+  U: "units",
+  P: "people",
+};
+
+// Handles scrolling down to element on key press
+const onKeyPress = (event: KeyboardEvent) => {
+  const shortcutKey = event.key.toUpperCase();
+  const elementId =
+    shortcutKeyLookup[shortcutKey as keyof typeof shortcutKeyLookup];
```

Comment:
this was a typescript TIL for me, just doing `const elementId = shortcutKeyLookup[shortcutKey]` was telling me "No index signature with a parameter of type 'string' was found on type"  https://bobbyhadz.com/blog/typescript-no-index-signature-with-parameter-of-type-string

Reply:
yep, these array accessors can be quite annoying in TS. although this works, it is still a type assertion that's shortcutting the type safety in the slightest way. it's totally tolerable here, because we know that `document.getElementById(elementId);` can handle when `elementId` is undefined (it returns `null`).

we _could_ add a proper type guard here to avoid the type assertion, but it would require a lot of code. 

TLDR, i like what you've got here, but maybe add a comment that acknowledges the type assertion?

Reply:
added a comment, thank you! yeah i have a hard time knowing when its okay/worth the tradeoff to shortcut type safety, glad you think this is alright and its a good thing to keep in mind